executor/generic.go

```diff
@@ -108,6 +108,14 @@ func (g *generic) GetRepository() string {
 	return g.Repository
 }
 
+func (g *generic) GetNamespace() string {
+	return g.Obj.GetMetaObject().GetNamespace()
+}
+
+func (g *generic) GetType() string {
+	return g.Obj.GetType()
+}
```

Comment:
I don't exactly know what this "GetType" means. What type of type are we talking about? Go types?

Reply:
it's the type of the CDRs and can have these string values: backup, check, archive, prune and restore.

Reply:
Maybe `GetJobType()` would be a clearer name?

Reply:
that sounds better to me, thanks

Reply:
I'll also update the GetNamespace to GetJobNamespace, if that's ok